core/sugarConstructors.ml

```diff
@@ -129,7 +133,7 @@ module SugarConstructors (Position : Pos)
 
   (** Fieldspec *)
 
-  let present        = Datatype.Present (Sugartypes.with_dummy_pos Datatype.Unit)
+  let present        = Datatype.Present (With_pos.make Datatype.Unit)
```

Comment:
Somehow I find the name `make` a bit uninformative in this case. Previously the intention to attach a dummy position was clear.

Reply:
Hmm suggestions for alternatives? Maybe just `of`?

Reply:
How about `WithPos.dummy`?

Reply:
Currently the `make` function takes an optional argument `pos` for the position. So it isn't just a function for dummy positions. Or would you prefer this not to be the case?

Reply:
> So it isn't just a function for dummy positions. 

I know - that's why I said that the intention to attach a dummy position is not as clear as it was before. When I was introducing `with_pos` data type I had the same dilemma, i.e. whether to have a separate function for dummy positions or just use an optional argument. I went with the former because the burden of using named arguments seemed a bit to heavy syntactically. But that's just one person's opinion.

Reply:
So yes, I would prefer to have a separate function to attach a dummy position.